src/test/kotlin/uk/gov/justice/digital/assessments/controller/AssessmentControllerTest.kt

```diff
@@ -10,34 +10,47 @@ import java.time.LocalDateTime
 
 @AutoConfigureWebTestClient
 class AssessmentControllerTest : IntegrationTest() {
-
     private val supervisionId = "SUPERVISION1"
 
     @Test
     fun `access forbidden when no authority`() {
-
         webTestClient.get().uri("/assessments/supervision/$supervisionId")
                 .header("Content-Type", "application/json")
                 .exchange()
                 .expectStatus().isUnauthorized
     }
 
-
     @Test
     fun `create a new assessment returns assessment`() {
+        val assessment = createAssessment("SupervisionId")
 
-        webTestClient.post().uri("/assessments/supervision")
-                .bodyValue(CreateAssessmentDto("SupervisionId"))
+        assertThat(assessment.assessmentId).isEqualTo(1)
```

Comment:
So here Stuart has done the traditional java testing with multiple asserts, but we had started using the kotlin data class type to assert a compared valid object with the returned object, which I quite like.

Some examples here https://phauer.com/2018/best-practices-unit-testing-kotlin/#utilize-data-classes

Reply:
I did try isEqualTo with the data class but the AssertJ failure messages didn't break down the fields. Happy to plug another assertion library in though.